debian/watch

```diff
@@ -0,0 +1,8 @@
+# See uscan(1) for format
```

Comment:
i actually find this comment useful, i always forget the `watch -> uscan` mapping, and i have many many years doing debian packaging.